.gitignore

```diff
@@ -5,4 +5,4 @@ archive/*
 build/*
 downloads/*
 diff/*
-dist/*
+Python-*-*-support.*.tar.gz
```

Comment:
I can put tarballs back to dist/ but .framework need to be in build/<os> so that Python build finds them.